src/ui/server/helpers/summary-lists/eligibility/eligibility-fields/index.ts

```diff
@@ -22,6 +23,24 @@ const {
 
 const { COMPANY_NUMBER, COMPANY_NAME } = COMPANIES_HOUSE;
 
+const {
+  ELIGIBILITY: {
+    EXPORTER_LOCATION_CHANGE,
+    COMPANIES_HOUSE_NUMBER_CHANGE,
+    ENTER_COMPANIES_HOUSE_NUMBER_CHANGE,
+    BUYER_COUNTRY_CHANGE,
+    TOTAL_VALUE_INSURED_CHANGE,
+    COVER_PERIOD_CHANGE,
+    UK_GOODS_OR_SERVICES_CHANGE,
+    END_BUYER_CHANGE,
+  },
+} = INSURANCE_ROUTES;
+
+const isCheckAndChange = false;
+const referenceNumber = undefined;
+const checkAndChangeLink = '';
+const isEligibility = true;
```

Comment:
Should this be `isInsuranceEligibility`? The changeLink function has it called this but in the eligibility summary lists its called `isEligibility`?